content/stratisd-rootfs-packaging.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "Packaging for stratisd 2.4.1"
+date = 2021-05-18
+weight = 19
+template = "page.html"
+render = true
++++
+
+*mulhern, Stratis Team*
+
+For Fedora packaging, we have decided to split out the dracut support into
+a separate subpackage, `stratisd-dracut`. This package must be installed
+in order to support booting from a Stratis filesystem. All other
+functionality is included in the `stratisd` package.
+
+The motivation for this change is to allow users greater flexibility and
+robustness. We understand that some users may choose to use Stratis but not to
+use Stratis for their root filesystem. These users may choose to install only
+the `stratisd` package.
+
+Other users may prefer to use a Stratis root filesystem. They should install
+the `stratisd-dracut` package, which has a hard dependency on the `stratisd`
+package. The `stratisd-dracut` package also includes a hard dependency on
+`dracut` itself and on `plymouth`. `plymouth` is used in order to obtain a
+password to unlock an encrypted Stratis root filesystem. Please consult
+[Stratis filesystem as the root filesystem] for further information about
+Stratis support for a root filesystem.
+
+We decided to implement this division due to a problem which would ensue
+if stratisd was installed but plymouth was not. In that case, the dracut
+module would not be installed in the initrd, and a subsequent reboot, when
+using a Stratis root filesystem, would fail.
```

Comment:
I would recommend "In that case, the regeneration of the initramfs on kernel updates would fail and render the system unbootable with the new kernel."